common/src/main/java/com/microsoft/identity/common/internal/ui/webview/AzureActiveDirectoryWebViewClient.java

```diff
@@ -223,7 +223,15 @@ private boolean processWebsiteRequest(@NonNull final WebView view, @NonNull fina
             final Context applicationContext = getActivity().getApplicationContext();
 
             // If CP is installed, redirect to CP.
-            if (!packageHelper.isPackageInstalledAndEnabled(
+            // TODO: Until we get a signal from eSTS that CP is the MDM app, we cannot assume that.
+            //       CP is currently working on this.
+            //       Until that comes, we'll only handle this in ipphone.
+            if (packageHelper.isPackageInstalledAndEnabled(
+                    applicationContext,
+                    AuthenticationConstants.Broker.IPPHONE_APP_PACKAGE_NAME) &&
```

Comment:
nit: if this became local ipPhonePackage, and ipPhoneSignature, say, this statement would be easier to read.